locales/fr_FR/pcsx2_Main.po

```diff
@@ -421,14 +420,12 @@ msgstr ""
 "dossier choisi."
 
 #: pcsx2/gui/AppConfig.cpp:976
-#, fuzzy
 msgid "Safest (No hacks)"
-msgstr "Sûr (rapide)"
+msgstr "Plus sûr (Pas de speedhacks)"
```

Comment:
Le plus sûr (Pas de speedhacks)
Wouldnt that fit better?

Reply:
Could work, but in the emulator point of view, I suppose that "Plus sûr" is better than using "Le plus sûr". Just my opinion.